packages/orama/tests/diacritics.test.ts

```diff
@@ -4,7 +4,7 @@ import { replaceDiacritics } from '../src/components/tokenizer/diacritics.js'
 t.test('Diacritics Replacer', t => {
   t.plan(1)
 
-  t.test('Should replace diacritics', t => {
+  t.test('should replace diacritics', t => {
```

Comment:
Is this a linter/prettier action?

Reply:
No: my OCD kicked in. Lol. Is that an issue?

Reply:
Nope, just wondering